src/nbl/video/CVulkanPhysicalDevice.cpp

```diff
@@ -1823,10 +1866,13 @@ core::smart_refctd_ptr<ILogicalDevice> CVulkanPhysicalDevice::createLogicalDevic
         rasterizationOrderAttachmentAccessFeatures.rasterizationOrderStencilAttachmentAccess = enabledFeatures.rasterizationOrderStencilAttachmentAccess;
 
         //colorWriteEnableFeatures [LIMIT SO ENABLE EVERYTHING BY DEFAULT]
-#if 0
-        cooperativeMatrixFeatures.cooperativeMatrix = true;
-        cooperativeMatrixFeatures.cooperativeMatrixRobustBufferAccess = enabledFeatures.cooperativeMatrixRobustBufferAccess;
-#endif
+
+        if (limits.cooperativeMatrix) {
+            cooperativeMatrixFeatures.cooperativeMatrixRobustBufferAccess = enabledFeatures.cooperativeMatrixRobustBufferAccess;
+        }
+
+        maintenance5Features.maintenance5 = true;
+        graphicsPipelineLibraryFeatures.graphicsPipelineLibrary = maintenance5Features.maintenance5 == 0;
```

Comment:
erm but you just set maintenance5 to true, maybe you should incorporate the feature struct bool setting into the ig statements at https://github.com/Devsh-Graphics-Programming/Nabla/pull/829/files#r1947966841

Reply:
`maintenance5Features.maintenance5 == 0` will never kick in because you're setting it to true on the line immediately above